serve/graph/schema/filter/transaction_filter.graphql

```diff
@@ -0,0 +1,283 @@
+  """
+  to_address: String
+
+  """
+  the denomination and amount of fund sent ("<amount><denomination>").
+  ex) `1000000ugnot`
+  """
+  amount: AmountInput
+}
+
+"""
+`TransactionVmMessageInput` represents input parameters required when the message router is `vm`.
+"""
+input TransactionVmMessageInput {
+  """
+  `MsgCallInput` represents input parameters required when the message type is `exec`.
+  """
+  exec: MsgCallInput
+
+  """
+  `MsgAddPackageInput` represents input parameters required when the message type is `add_package`.
+  """
+  add_package: MsgAddPackageInput
+
+  """
+  `MsgRunInput` represents input parameters required when the message type is `run`.
+  """
+  run: MsgRunInput
+}
+
+"""
+`MsgCallInput` represents input parameters required when the message type is `exec`.
+"""
+input MsgCallInput {
+  """
+  the bech32 address of the function caller.
+  You can filter by the function caller's address.
+  ex) `g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5`
+  """
+  caller: String
+
+  """
+  the amount of funds to be deposited to the package, if any ("<amount><denomination>").
+  ex) `1000000ugnot`
+  """
+  send: AmountInput
+
+  """
+  the gno package path.
+  """
+  pkg_path: String
+
+  """
+  the function name being invoked.
+  """
+  func: String
+
+  """
+  `args` are the arguments passed to the executed function.
+  The arguments are checked in the order of the argument array and
+  if they are empty strings, they are excluded from the filtering criteria.
+  ex) `["", "", "1"]` <- Empty strings skip the condition.
+  """
+  args: [String!]
+}
+
+"""
+`MsgAddPackageInput` represents input parameters required when the message type is `add_package`.
+"""
+input MsgAddPackageInput {
+  """
+  the bech32 address of the package deployer.
+  You can filter by the package deployer's address.
+  ex) `g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5`
+  """
+  creator: String
+
+  """
+  the package being deployed.
+  """
+  package: MemPackageInput
+
+  """
+  the amount of funds to be deposited at deployment, if any ("<amount><denomination>").
+  ex) `1000000ugnot`
+  """
+  deposit: AmountInput
+}
+
+"""
+`MsgRunInput` represents input parameters required when the message type is `run`.
+"""
+input MsgRunInput {
+  """
+  the bech32 address of the function caller.
+  You can filter by the function caller's address.
+  ex) `g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5`
+  """
+  caller: String
+
+  """
+  the amount of funds to be deposited to the package, if any ("<amount><denomination>").
+  ex) `1000000ugnot`
+  """
+  send: AmountInput
+
+  """
+  the package being executed.
+  """
+  package: MemPackageInput
+}
+
+"""
+`MemPackageInput` represents a package stored in memory.
+"""
+input MemPackageInput {
+  """
+  the name of the package.
+  """
+  Name: String
+
+  """
+  the gno path of the package.
+  """
+  Path: String
+
+  """
+  the associated package gno source.
+  """
+  Files: [MemFileInput]
+}
```

Comment:
```suggestion
input MemPackageInput {
  """
  the name of the package.
  """
  name: String

  """
  the gno path of the package.
  """
  path: String

  """
  the associated package gno source.
  """
  files: [MemFileInput]
}
```

Reply:
The conventions for MemPackage, MemFile followed gno's json types.
The actual json response comes in Pascalcase.
https://github.com/gnolang/gno/blob/831bb6f92e1a2217242169dab1f4fd1f87e5eaa0/tm2/pkg/std/std.proto#L18-L27

Do you think that when an indexer shows a transaction, it should conform to the indexer's convention? 
Or is there a possibility to unify the convention in the chain?